org.opent2t.sample.hub.superpopular/com.nest.hub/js/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "opent2t-translator-com-nest-hub",
-  "version": "1.0.2",
+  "version": "0.2.0",
```

Comment:
I thought Avneet published everything (except Wink) under version 0.1.0. I will double check with Avneet.

---
In reply to: [98255520](https://github.com/openT2T/translators/pull/146#discussion_r98255520) [](ancestors = 98255520)